integrations/gcp/.port/spec.yaml

```diff
@@ -0,0 +1,26 @@
+type: gcp
+description: gcp integration for Port Ocean
+icon: Cookiecutter # Should be one of the available icons in Port
+features:
+  - type: exporter
+    section: Cloud Providers
+configurations:
+  - name: appHost
+    required: false
+    type: url
+    description: The host of the Port Ocean app.
```

Comment:
redundant

Reply:
What is redundant? this variable or the description?

Reply:
Give a better description 
The description is basically saying the name in other words